### DISCUSSION TITLE
Message between fragments in PIE model

### DISCUSSION THREAD

**Discussion by karimDev13** [alibaba/GraphScope#2108] [Q&A]
Hi, how can i access to the boundary nodes from the function IncEVAL( ) from fragment F ? and what's the form of the message between fragments ?

**Comment by yecol** [comment]
Hi @karimDev13, 
Thank you for your interest in GraphScope!
For your question, please checkout the SSSP example code,
https://github.com/alibaba/libgrape-lite/blob/master/examples/analytical_apps/sssp/sssp.h#L152
